src/common/InteractUtils.cpp

```diff
@@ -3,6 +3,7 @@
 //
 
 #include "InteractUtils.h"
+#define MAXITEMS 255
```

Comment:
use `const uinrt8_t` instead of #define macro
http://www.stroustrup.com/bs_faq2.html#macro

Reply:
Good point, done.